### ISSUE TITLE
[A11y]On 'T' key press in scan mode, Narrator is reading out every row in the same table

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7892]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1069968
Screen Reader: 
Narrator
​
Repro Steps:​
1.Start Screen Reader. Hit the URL "https://www.nuget.org"
2.Tab till "Packages" link in the top navigation and press enter​
3.Tab till any existing package and press enter​
4.Tab till "GitHub Usage" section and press enter​ to expand it
5.Turn on scan mode for Narrator using Caps Lock+ Space bar
6.Using 'T' key, navigate to different tables on the page
7.Check if the Narrator is navigating to different rows of the same table on 'T' key press
​
Actual Result:​
On 'T' key press in scan mode, Narrator is reading out every row in the same table

NOTE: NVDA is working as expected(added attachment for reference)

Expected Result:​
On 'T' key press in scan mode, Narrator focus should navigate to different tables of the page and not to different rows in same table

Notes:
This seems related to https://github.com/NuGet/NuGetGallery/issues/7891 in the narrator is seeing tables where there are not any.
I suspect that the rows in the GitHub usage section are being read as tables, either from layout or explicit tags. We should modify this section to look more like the version table below it.

**Comment by ryuyu** [comment]
Unable to repro. Narrator must have pushed a fix for this very recently? I was unable to repro any of the scan mode issues.
